nms/app/state/lte/SubscriberState.ts

```diff
@@ -102,12 +102,16 @@ export async function FetchSubscriberState(props: FetchProps) {
   const {networkId, enqueueSnackbar, id} = props;
   if (id !== null && id !== undefined) {
     try {
-      return (
-        await MagmaAPI.subscribers.lteNetworkIdSubscriberStateSubscriberIdGet({
-          networkId,
-          subscriberId: id,
-        })
-      ).data;
+      return {
+        [id]: (
+          await MagmaAPI.subscribers.lteNetworkIdSubscriberStateSubscriberIdGet(
```

Comment:
If the response it empty this will result in a state like:
``` { IMSI1234 : {} }```
but when fetching all subscriber states this imsi will not be included.
This would only make a difference if we iterate over the keys though.